### PR TITLE
fix app build

### PR DESCRIPTION
**Pull request by smitev**

### CODE REVIEW

File: config-overrides.js [hunk 1]
```diff
@@ -25,11 +25,11 @@ const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
 
 module.exports = function override(config, env) {
   config.resolve = {
+    ...config.resolve,
     fallback: {
+      ...config.resolve.fallback,
       stream: require.resolve('stream-browserify'),
     },
-    // NOTE: I added this; missing in Substrate template and causes missing module errors;
-    extensions: [...config.resolve.extensions, '.ts', '.js'],
```
Comment (smitev) [inline config-overrides.js:32]: You need first to destruct the existing "resolve" properties before adding the fallback. That way you don't need to add "extensions".

File: config-overrides.js [hunk 2]
```diff
@@ -25,11 +25,11 @@ const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
```
Comment (slaveatanasov) [inline config-overrides.js:25]: Please git pull from develop branch and discard these changes. I got it working in develop branch.

File: package.json [hunk 3]
```diff
@@ -4,12 +4,12 @@
   "version": "3.4.0",
```
Comment (slaveatanasov) [inline package.json:4]: This one is OK.

**Comment by smitev** [comment]
implemented in other PR

**Pull request closed without merging**